packages/frontend/src/components/ChartConfigPanel/FieldLayoutOptions.tsx

```diff
@@ -156,28 +157,33 @@ const FieldLayoutOptions: FC<Props> = ({ items }) => {
                     </Button>
                 )}
             </AxisGroup>
-            <AxisGroup>
-                <AxisTitle>Group</AxisTitle>
-                <AxisFieldDropdown>
-                    <FieldAutoComplete
-                        fields={availableDimensions}
-                        placeholder="Select a field to group by"
-                        activeField={groupSelectedField}
-                        onChange={(item) => {
-                            setPivotDimensions([getItemId(item)]);
-                        }}
-                    />
-                    {groupSelectedField && (
-                        <DeleteFieldButton
-                            minimal
-                            icon="cross"
-                            onClick={() => {
-                                setPivotDimensions([]);
+            <BlockTooltip
+                content="You need at least one metric in your chart to add a group"
+                disabled={chartInvolvesMetrics}
+            >
+                <AxisGroup disabled={!chartInvolvesMetrics}>
```

Comment:
Let's apply the disabled in the `<FieldAutoComplete />` instead to keep our UI consistent. 
Example of disabled inputs
![Screenshot 2022-09-16 at 17 03 10](https://user-images.githubusercontent.com/9117144/190682179-f4e1700c-f00f-4b01-bcdc-1af2ef047abd.png)